dev/buildtool/git_support.py

```diff
@@ -863,7 +863,7 @@ def find_newest_tag_and_common_commit_from_id(self, git_dir, commit_id):
         if tag_commit == commit_id:
             logging.debug("Commit %s is already tagged with %s", tag_commit, tag)
         else:
-            logging.warning(
```

Comment:
This was previously an important warning when building a BOM without `tag_branch` command help.
However `tag_branch` calls this method and based on the result will generate a new tag or not, taking care of the situation for us.